client/src/components/game/ruler/Ruler.vue

```diff
@@ -43,10 +44,35 @@
           </span>
       </div>
     </div>
+    <div class="row bg-primary pt-2 pb-2">
+      <div class="col-6">
+      Speed Mod.
+      </div>
+      <div class="col-6 text-right">
+        <select class="form-control form-control-sm" v-model="speedModifier" @change="onSpeedModifierChanged">
+          <option value="1">Normal (1.0x)</option>
+          <option value="0.5">0.5x</option>
+          <option value="1.5">1.5x</option>
+          <option value="2.0">2.0x</option>
```

Comment:
Are these **all** of the possible speed modifiers?

It may be a better approach to dynamically create this option list for all possible types. Something like a `v-for` on an array of speed modifiers generated from the carrier spec list.

If so it would be better placed in its own component.